src/components/sideBar/SideBar.jsx

```diff
@@ -40,7 +48,7 @@ const SideBar = ({ onToggleSideBar, handleToggleSideBar }) => {
         <span>Dislikes</span>
       </li>
       <hr />
-      <li>
+      <li onClick={logoutHandler}>
```

Comment:
I would do handleLogout

Reply:
I am using this convention from starting